sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizePiiEntitiesTests.cs

```diff
@@ -13,7 +13,7 @@ public class RecognizePiiEntitiesTests : TextAnalyticsClientLiveTestBase
     {
         public RecognizePiiEntitiesTests(bool isAsync) : base(isAsync) { }
 
-        private const string EnglishDocument1 = "A developer with SSN 859-98-0987 whose phone number is 800-102-1100 is building tools with our APIs.";
+        private const string EnglishDocument1 = "A developer with SSN 859-98-0987 whose phone number is 800-102-1100 is building tools with our APIs. They work at Microsoft";
```

Comment:
I modified the input so we could have another entity returned and have more test scenarios